app/components/admin/custom_fields/hierarchy/change_item_parent_dialog_component.rb

```diff
@@ -48,17 +48,48 @@ def form_id = "custom-fields--change-item-parent-form"
 
         def form_arguments
           {
-            form_id:,
+            id: form_id,
             url: change_parent_custom_field_item_path(custom_field_id: @custom_field.id, id: @hierarchy_item.id),
             model: form_model,
             method: :post
           }
         end
 
+        def hierarchy_items
+          hashed_hierarchy = @custom_field.hierarchy_root.hash_tree
+          hashed_hierarchy.nil? ? {} : hashed_hierarchy.first[1]
+        end
+
+        def add_sub_tree(tree, hierarchy_hash)
+          hierarchy_hash.each do |item, child_hash|
+            if child_hash.empty?
+              tree.with_leaf(**item_options(item))
+            else
+              expanded = current?(item) || child_hash.any? { |child, _| current?(child) }
+
+              tree.with_sub_tree(expanded: expanded, **item_options(item)) do |sub_tree|
```

Comment:
Where is Rubocop?

:policeman: 

```suggestion
              tree.with_sub_tree(expanded:, **item_options(item)) do |sub_tree|
```